### PR TITLE
Update change log for beta-1.

### PR DESCRIPTION
**Pull request by xfxyjwf**
@pherl to review

### CODE REVIEW

**Comment by xfxyjwf** [comment]
@haberman to help take a look at the wording/grammar.

File: CHANGES.txt [hunk 1]
```diff
@@ -1,3 +1,109 @@
+2015-08-26 version 3.0.0-beta-1 (C++/Java/Python/Ruby/Nano/Objective-C/C#)
+  About Beta
+  * This is the first beta release of protobuf v3.0.0. Not all languages
+    reaches beta stage. Except for those languages marked as beta, others
```
Comment (haberman) [inline CHANGES.txt]: Not all languages have reached beta stage.  Languages not marked as beta are still in alpha.

File: CHANGES.txt [hunk 2]
```diff
@@ -1,3 +1,109 @@
...
+
+  General
+  * Proto3 JSON is supported in several languages (fully supported in C++
+    and Java, partially supported in Ruby/C#). The JSON spec is defined in
+    the proto3 language guide:
+
+      https://developers.google.com/protocol-buffers/docs/proto3#json
+
+    We will publish a more detailed spec to define the exact behavior of
+    proto3 conformant JSON serializers and parsers. Before then, do not rely
```
Comment (haberman) [inline CHANGES.txt]: proto3-conformant

Until then

File: CHANGES.txt [hunk 3]
```diff
@@ -1,3 +1,109 @@
...
+    on specific behaviors of the implementation if it’s not documented in
+    the above spec. More specifically, the behavior is not yet finalized for
+    the following:
+      - Parsing invalid JSON input (e.g., input with trailing commas).
+      - Non-camelCase names in JSON input
```
Comment (haberman) [inline CHANGES.txt]: end with period to match

File: CHANGES.txt [hunk 4]
```diff
@@ -1,3 +1,109 @@
...
+
+  Ruby (Alpha)
+  * Many bugfixes:
+      - fixed parsing/serialization of bytes, sint, sfixed types
+      - other parser bugfixes
+      - fixed memory leak affecting Ruby 2.2
+
+  JavaNano (Alpha)
+  * JavaNano generated code now will be put in a nano package by default to
+    avoid confliction with Java generated code.
```
Comment (haberman) [inline CHANGES.txt]: conflicts

File: CHANGES.txt [hunk 5]
```diff
@@ -1,3 +1,109 @@
...
+
+  Objective-C (Alpha)
+  * Added nonnil markup to ObjC library. Require SDK 8.4+ to build.
```
Comment (haberman) [inline CHANGES.txt]: non-nil (I think?)

Requires

File: CHANGES.txt [hunk 6]
```diff
@@ -1,3 +1,109 @@
...
+  * Many bugfixes:
+      - Removed the class/enum filter.
+      - Renamed some internal types to avoid conflicts with the well know types
```
Comment (haberman) [inline CHANGES.txt]: well-known

File: CHANGES.txt [hunk 7]
```diff
@@ -1,3 +1,109 @@
...
+        protos.
+      - Added missing support for parsing repeated primitive fields in packed or
+        unpacked forms.
+      - Added *Count for repeated and map<> fields to avoid auto create when
```
Comment (haberman) [inline CHANGES.txt]: auto-create

**Comment by pherl** [comment]
LGTM

File: CHANGES.txt [hunk 8]
```diff
@@ -1,3 +1,109 @@
...
+        checking for them being set.
+
+  C# (Alpha)
+  * Namespace changed to Google.Protobuf (and NuGet package will be named
+    correspondingly).
+  * Target platforms now .NET 4.5 and selected portable subsets only
+    Removed lite runtime.
```
Comment (haberman) [inline CHANGES.txt]: Does this need its own bullet? And period on the end of the previous line?

**Comment by haberman** [comment]
Done with my comments. :)

File: CHANGES.txt (hunk 1 above)
Comment (xfxyjwf) [inline CHANGES.txt]: Done.

File: CHANGES.txt (hunk 2 above)
Comment (xfxyjwf) [inline CHANGES.txt]: Done.

File: CHANGES.txt (hunk 3 above)
Comment (xfxyjwf) [inline CHANGES.txt]: Done.

File: CHANGES.txt (hunk 4 above)
Comment (xfxyjwf) [inline CHANGES.txt]: Done.

File: CHANGES.txt (hunk 5 above)
Comment (xfxyjwf) [inline CHANGES.txt]: Done.

Changed nonnil to non-null. The exact markup is NS_ASSUME_NONNULL_BEGIN/END

File: CHANGES.txt (hunk 6 above)
Comment (xfxyjwf) [inline CHANGES.txt]: done

File: CHANGES.txt (hunk 7 above)
Comment (xfxyjwf) [inline CHANGES.txt]: done

**Comment by xfxyjwf** [comment]
Addressed all comments. PTAL.

**Comment by haberman** [comment]
LGTM.

**Pull request merged**